app/models/profile.rb

```diff
@@ -49,6 +49,13 @@ def activate
 
   def activate_after_passing_review
     update!(fraud_review_pending: false, fraud_rejection: false)
+    fraud_event = user.fraud_event
+    irs_attempts_api_tracker&.fraud_review_adjudicated(
+      decision: 'pass',
+      fraud_event_id: fraud_event&.id,
```

Comment:
in general I think it's a bad pattern to expose our internal DB ids in any API. I would add a `uuid` column (and use our `NonNullUuid` concern)

Reply:
I like that, if we're going to keep this.

There's a question of whether we actually need this parameter. It sounds like the client wants session IDs. It seems like this extra detail we have for free may help make things easier, in which case I'm on board with making it a UUID. But I just want to call out that this is only being included because it felt like a good idea to me, not because the requirements actually call for an identifier.

Reply:
oh! in that case, let's drop from the event it if there is no business case for it (maybe add the UUID column just in case since we can't go back in time and add it for the past)